mne_neuron/network.py

```diff
@@ -333,11 +338,13 @@ def _create_all_src(self):
     # nc = pc.gid_connect(source_gid, target_syn), weight,delay
     # Both for synapses AND for external inputs
     def __parnet_connect(self):
+        from . import sim
```

Comment:
do you need to have the import nested in order for it to work? Otherwise, you could move the import outside. Even better if you do:

```py
from .sim import pc
```

then the existing code would stay as it is without having to modify every line